### PR TITLE
Dpf 20

### PR DESCRIPTION
**Pull request by zahoor-moj**

### CODE REVIEW

**Comment by zahoor-moj** [comment]
DPD only contains hospital movements. Ready to promotoe to test/preprod

File: prisons/orphanage/dpd001-hospital-and-court-movements-1.0.3.json [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+      "id": "datamart",
+      "name": "datamart"
+    }
+  ],
+  "dataset": [
+    {
+      "id": "court-dataset",
+      "name": "court-dataset",
+      "datasource": "datamart",
+      "query": "SELECT distinct prisoners.number AS prison_number,name AS name,movements.time AS date,movements.direction as direction,movements.type as type,movements.origin as origin,movements.origin_code as origin_code,movements.destination as destination,movements.destination_code as destination_code,movements.reason FROM datamart.domain.movement_movement as movements JOIN datamart.domain.prisoner_prisoner as prisoners ON movements.prisoner = prisoners.id JOIN datamart.domain.prisoner_status as status on status.id = prisoners.id Where type='CRT'  and lower(direction) = 'out' and status.in_out_status ='OUT' and status.active = 'Y'",
```
Comment (stalwartsolutions) [inline prisons/orphanage/dpd001-hospital-and-court-movements-1.0.3.json:21]: Is the `distinct` necessary here? Are there any duplicates?

File: src/main/resources/definitions/prisons/test/test-report.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
       "id": "test",
       "name": "Test data",
       "datasource": "redshift",
-      "query": "SELECT 'One' as column1, 2 as column2, 'Three' as column3, TO_TIMESTAMP('2004-05-06T07:08:09Z', 'YYYY-MM-DDTHH24:MI:ssZ') as column4 UNION SELECT 'Remove' as column1, null as column2, null as column3, null as column4 UNION SELECT 'Eleven' as column1, 12 as column2, 'Thirteen' as column3, TO_TIMESTAMP('2014-12-16T17:18:19Z', 'YYYY-MM-DDTHH24:MI:ssZ') as column4",
+      "query": "SELECT 'One' as column1, 2 as column2, 'Three' as column3, TO_TIMESTAMP('2004-05-06 07:08:09', 'YYYY-MM-DD HH24:MI:ss') as column4 UNION SELECT 'Remove' as column1, null as column2, null as column3, null as column4 UNION SELECT 'Eleven' as column1, 12 as column2, 'Thirteen' as column3, TO_TIMESTAMP('2014-12-16 17:18:19', 'YYYY-MM-DD HH24:MI:ss') as column4",
       "schema": {
```
Comment (stalwartsolutions) [inline src/main/resources/definitions/prisons/test/test-report.json:22]: These changes look odd. This is the test report used for acceptance tests. Are these changes to support a format change or such?

File: src/main/resources/definitions/prisons/orphanage/dpd001-hospital-movements.json [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+            "formula": "",
+            "visible": true,
+            "sortable": true,
+            "defaultsort": false,
+            "filter": {
+              "type": "Radio",
+              "staticoptions": [
+                {
+                  "name": "In",
+                  "display": "IN"
```
Comment (stalwartsolutions) [inline src/main/resources/definitions/prisons/orphanage/dpd001-hospital-movements.json:129]: This seems like an odd combination - the `name` which is used by the back-end is in ProperCase, but the value displayed to the user is in uppercase.

I'd would have expected the `display` value to be closer to normal casing, and the `name` value to be lowercase (not that it matters much to the app).

**Comment by stalwartsolutions** [comment]
This PR looks like it's based off an out of date version of main. I expect that's why you've got some odd changes to the test report, and a validation failure.

**Comment by zahoor-moj** [comment]
Closing as part of tidying up.

**Pull request closed without merging**